csharp_interface/cozmoInterface/connection.cs

```diff
@@ -24,25 +24,23 @@ namespace Anki
 {
   namespace Cozmo
   {
-    public class Connection
+    public class SDKConnection
```

Comment:
I'm not sure if I like the rename here. We already have Connection in the namespace of `Anki::Cozmo`, so SDK doesn't really add anything to the name. On the python side I believe the equivalent is named `CozmoConnection`, which then if we're saying people might miss the namespace will at least say what the connection is for.

I still accepted the PR because it's up to you if you want to change the name.